### DISCUSSION TITLE
ssh to debian 10.4 minimal

### DISCUSSION THREAD

**Discussion by radoslavirha** [utmapp/UTM#2447] [UTM Help (macOS)]
Hi how can I ssh to my virtual machine (debian 10.4 minimal) from host system (MacBook Air m1)?... using 10.0.2.2  timeouts after some time. I use default network settings for Debian vm. I need it for development, maybe I'll forward some ports. Thanks

**Comment by hungtcs** [comment]
see https://github.com/utmapp/UTM/discussions/2535
